token/client/src/token.rs

```diff
@@ -514,6 +514,16 @@ where
         let payer_key = self.payer.pubkey();
         let fee_payer = Some(&payer_key);
 
+        {
+            // the most expensive instruction is VERIFY_TRANSFER_WITH_FEE_COMPUTE_UNITS
+            let requested_units = 500_000;
```

Comment:
Ideally transaction should be simulated to get required CU limits; or can manually count CUs while preparing token instructions. Setting a dummy number for now to test if this works.